src/vivarium/framework/population/population_view.py

```diff
@@ -60,11 +61,15 @@ def __init__(
         query
```

Comment:
Need to update the docstring for `column` to read "The set of columns this view should have access to. If requires_all_columns is True, this is instead the columns created by the component containing this population view."

This also applies to all the relevant locations in the manager and interface.